frontend/.env.example

```diff
@@ -0,0 +1 @@
+BACKEND_ENDPOINT=http://localhost:8080
```

Comment:
Dunno if we care but no new line on this & a few other files. Will approve if this is the last comment I don't think it matters too much just raising the spot

Reply:
nah I am a fan of new line at end of files, will see if it is easy to add via eslint

Reply:
okay I've added a eslint plugin that does this for all of our js, ts, tsx files, and have manually done the rest. Can switch out plugins later if we want something else